### PR TITLE
[stable/fairwds-insights] Updated cronjob close-tickets to update-tickets

### PR DESCRIPTION
**Pull request by jdesouza**
**Why This PR?**
_a short description of why this PR is needed_

Fixes #

**Changes**
Changes proposed in this pull request:

*
*

**Checklist:**

* [x] I have included the name of the chart in the title of this PR in square brackets i.e. `[stable/goldilocks]`.
* [x] I have updated the chart version in `Chart.yaml` following Semantic Versioning.
* [x] Any new values are backwards compatible and/or have sensible default.
* [x] Any new values have been added to the README for the Chart, or `helm-docs --sort-values-order=file` has been run for the charts that support it.

### CODE REVIEW

File: stable/fairwinds-insights/ci/test-values.yaml [hunk 1]
```diff
@@ -65,7 +65,7 @@ resourcesRecommendationsCronjob:
   enabled: true
   schedules: []
 
-closeTicketsCronjob:
+updateTicketsCronjob:
```
Comment (rbren) [inline stable/fairwinds-insights/ci/test-values.yaml]: huh this isn't doing anything anymore--we should remove these values and replace them with the new format `cronjobs.update-tickets`

File: stable/fairwinds-insights/Chart.yaml [hunk 2]
```diff
@@ -2,7 +2,7 @@ apiVersion: v2
 appVersion: "15.3"
```
Comment (rbren) [inline stable/fairwinds-insights/Chart.yaml]: Let's hold off on this PR until we update appVersion to 15.4

Comment (jdesouza) [reply stable/fairwinds-insights/Chart.yaml]: removed

File: stable/fairwinds-insights/ci/test-values.yaml (hunk 1 above)
Comment (jdesouza) [reply stable/fairwinds-insights/ci/test-values.yaml]: Removed

**Review by rbren — APPROVED** [review]

**Pull request merged**